grafana/overlays/emea/jerry/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: opf-monitoring
+generatorOptions:
+  disableNameSuffixHash: true
+configMapGenerator:
+  - name: grafana-config-overrides
+    literals:
+      - GF_AUTH_GENERIC_OAUTH_ROLE_ATTRIBUTE_STRICT=true
+resources:
+  - ../../../base
```

Comment:
dont you need to include this kustomization in the jerry deployments?
And I don't know if we have the grafana operator deployed on jerry